package.json

```diff
@@ -197,7 +197,7 @@
   "dependencies": {
     "@grpc/grpc-js": "^1.9.4",
     "@grpc/proto-loader": "^0.7.5",
-    "@newrelic/security-agent": "^1.3.0",
+    "@newrelic/security-agent": "^1.5.0",
```

Comment:
this is not needed. a fresh install of agent will install 1.5.0 due to the caret matching on the dependency.  This is a duplicate of #2470 which we closed as not needed